rollup.config.js

```diff
@@ -1,21 +1,23 @@
-import nodeResolve from '@rollup/plugin-node-resolve';
-import replace from '@rollup/plugin-replace';
-import commonjs from '@rollup/plugin-commonjs';
-import babel from '@rollup/plugin-babel';
-import json from '@rollup/plugin-json';
-import typescript2 from 'rollup-plugin-typescript2';
-import { terser } from 'rollup-plugin-terser';
-import sourceMaps from 'rollup-plugin-sourcemaps';
-import pkg from './package.json';
+import nodeResolve from "@rollup/plugin-node-resolve";
```

Comment:
Should this be a ts file or a js file? Not sure. Just wondering.

Reply:
Do you mean this `rollup.config.js` file? I found they only started to support typescript config file in recent release: https://github.com/rollup/rollup/releases/tag/v2.52.0

Reply:
Yeah. Perhaps we can change it to `ts` too.